src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -126,7 +128,7 @@ private void testContent() throws Exception {
         homePage.verifyHtmlMainContent("/instructorHomeNewInstructorWithoutSampleCourse.html");
 
         CourseAttributes newCourse = CourseAttributes
-                .builder("newIns.wit-demo", "Sample Course 101", "UTC")
+                .builder("newIns.wit-demo", "Sample Course 101", ZoneId.of("UTC"))
```

Comment:
Same as above for `Const.DEFAULT_TIMEZONE`.

Reply:
This is a test. It should explicitly states what timezone it's using